moped-editor/src/views/projects/projectView/ProjectComponents/utils/form.js

```diff
@@ -117,12 +155,57 @@ export const useInitialValuesOnAttributesEdit = (
     setValue("subcomponents", selectedSubcomponents);
   }, [subcomponentOptions, initialFormValues, setValue]);
 
+  // Set the selected phase after the phase options are loaded
+  useEffect(() => {
```

Comment:
Thanks for hanging in there with the pattern used to set these values. 🙏 This is definitely ready for a refactor, and seeing more fields added to the form makes that super obvious!